web/assets/js/app.js

```diff
@@ -401,4 +401,23 @@ $(document).ready(function()
             btnTextTrue: 'Continue'
         });
     });
+
+    var validateDateFormat = function (value) {
+        var parts = value.split('/');
+        if (parts.length < 3) {
+            return false;
+        }
+        var dt = new Date(parts[2], parts[0] - 1, parts[1]);
+        return (dt && dt.getMonth() === parseInt(parts[0], 10) - 1 && dt.getFullYear() === parseInt(parts[2]));
+    };
+
+    window.Parsley.addValidator('dateMdy', {
+        validateString: function (value) {
+            return validateDateFormat(value);
+        },
+        messages: {
+            en: 'Invalid date format.'
+        },
+        priority: 32
+    });
```

Comment:
We have an existing `parsley-comparison.js` file with some custom validators. Would it make sense to rename that file to some thing more generic (maybe `parsley-validators.js`) and then add this code there?